recipes/r-mess/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.5.4' %}
```

Comment:
conda-forge?

Reply:
It's from the wrapper script, yes.

Reply:
No I mean should this go to conda-forge :)

Reply:
It has requirements in bioconda (r-kinship2), so no.

Reply:
Mh, this will bite us sooner or later. Another example where it would be better to move all r-* packages which have no bioconductor dependencies into conda-forge.